canonical/lxd-ui
# Use action button for a consistent loading button behaviour with vanilla guidelines

## Done

- replace all occurrences of submit button with action button, to have a consistent loading button behaviour with vanilla guidelines
- remove submit button component

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check occurrences of the submit button for visual regressions
